openapi/trireg2.yaml

```diff
@@ -2,18 +2,20 @@ openapi: "3.0.3"
 info:
   title: "TriReg2 API"
   description: "An OpenAPI specification for time registrations"
-  version: "0.0.5"
+  version: "0.0.6"
 servers:
-  - url: "http://0.0.0.0:8080"
+- url: "http://0.0.0.0:8080"
```

Comment:
is this indentation correct?

Reply:
Yes ( https://stackoverflow.com/a/71968274 ) for yaml sequences the indentation level doesn't matter. It looks like the VSC extension I have for yaml formatting prefers this style.